src/processTargets/marks/LineNumberStage.ts

```diff
@@ -35,7 +35,7 @@ const getLineNumber = (
     case "absolute":
       return lineNumber;
     case "relative":
-      return editor.selection.active.line + lineNumber;
+      return editor.selections[0].active.line + lineNumber;
```

Comment:
Huh.  This feels kinda VSCode-specific 🤔.  I wonder if relative line numbers should be part of the IDE abstraction.  Not sure.  We should prob discuss with maintainers of other sidecar impls to see what makes sense in their editors

Reply:
The whole notion of having a main selection is potentially vscode specific as well. We definitely need to have a discussion about this.